components/Players/TrackList.vue

```diff
@@ -2,9 +2,9 @@
   <v-simple-table dense class="track-table no-select">
     <thead>
       <tr>
-        <th style="width: 6em" class="pr-0 text-center" scope="col">#</th>
+        <th style="width: 4em" class="pr-0 text-center" scope="col">#</th>
         <th style="width: 3em" class="pr-0 pl-0" scope="col"></th>
-        <th scope="col">Title</th>
+        <th scope="col">{{ $t('trackTitle') }}</th>
```

Comment:
I think we should keep better organization of strings. I propose an structure like ``item.type.key``. So, in this case, it will be ``item.tracklist.trackTitle`` :P